osu.Framework/Testing/TestSceneTestRunner.cs

```diff
@@ -82,7 +83,7 @@ void complete()
                 {
                     AddInternal(test);
 
-                    Logger.Log($@"💨 {test} running");
+                    Logger.Log($@"💨 {test} {TestContext.CurrentContext.Test.Name}");
```

Comment:
This one I'm also not 100% on. It produces output like

```
💨 BindableAutoUnbinding(TestSceneBindableAutoUnbinding) TestBindableAutoUnbindingAssign
```

The first word is the class name without the `TestScene` prefix, the second is the full class name, and the last word is the method name. The first two words come from `Drawable.ToString()` overall. Maybe a particularly egregious example, but still pretty inscrutable. How about something like:

```suggestion
                    Logger.Log($@"💨 Class: {test.Name}");
                    Logger.Log($@"💨 Test:  {TestContext.CurrentContext.Test.Name}");
```

which produces

```
[runtime] 2021-12-23 10:10:58 [verbose]: 💨 Class: BindableAutoUnbinding
[runtime] 2021-12-23 10:10:58 [verbose]: 💨 Test:  TestBindableAutoUnbindingAssign
```

Reply:
Not a huge fan of the repeating icon, and couldn't decide on a second icon so left it on one line. Would either want whitespace or just use a large dash in its place. I'm not against split on two lines, though.

Reply:
Hmmm okay. How about a large diamond to match the small diamonds for the test step?

```
[runtime] 2021-12-23 10:28:09 [verbose]: 💨 Class: BindableAutoUnbinding
[runtime] 2021-12-23 10:28:09 [verbose]: 🔶 Test:  TestBindableAutoUnbindingAssign
[runtime] 2021-12-23 10:28:09 [verbose]: 🔸 Step #1 add drawables
[runtime] 2021-12-23 10:28:09 [verbose]: 🔸 Step #2 attempt value transfer
[runtime] 2021-12-23 10:28:09 [verbose]: 🔸 Step #3 transfer 1-2 completed
[runtime] 2021-12-23 10:28:09 [verbose]: 🔸 Step #4 transfer 1-3 completed
[runtime] 2021-12-23 10:28:09 [verbose]: 🔸 Step #5 transfer 1-4 completed
[runtime] 2021-12-23 10:28:09 [verbose]: 🔸 Step #6 expire child 4
[runtime] 2021-12-23 10:28:09 [verbose]: 🔸 Step #7 attempt value transfer
[runtime] 2021-12-23 10:28:09 [verbose]: 🔸 Step #8 transfer 1-2 completed
[runtime] 2021-12-23 10:28:09 [verbose]: 🔸 Step #9 transfer 1-3 completed
[runtime] 2021-12-23 10:28:09 [verbose]: 🔸 Step #10 transfer 1-4 skipped
[runtime] 2021-12-23 10:28:09 [verbose]: 🔸 Step #11 expire child 3
[runtime] 2021-12-23 10:28:09 [verbose]: 🔸 Step #12 attempt value transfer
[runtime] 2021-12-23 10:28:09 [verbose]: 🔸 Step #13 transfer 1-2 fails
[runtime] 2021-12-23 10:28:09 [verbose]: ✅ TestSceneBindableAutoUnbinding completed
```

Reply:
Seems okay.

Reply:
Have applied this, with one more small change (changed it so that the concrete type of the test scene is logged rather than the one with the `TestScene` prefix stripped, as it was looking pretty confusing for cases like `TestSceneTextBox`).